### DISCUSSION TITLE
U Toronto STA 4273 Winter 2021: Minimizing Expectations - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#507] [Announcements]
# U Toronto STA 4273 Winter 2021: Minimizing Expectations - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/STA4273/

**Comment by Jinhua-Si** [comment]
麻烦问下这课有视频吗
